src/System.Private.CoreLib/shared/System/Buffers/StandardFormat.cs

```diff
@@ -146,40 +146,47 @@ private static bool ParseHelper(ReadOnlySpan<char> format, out StandardFormat st
         /// </summary>
         public override string ToString()
         {
-            unsafe
+            Span<char> buffer = stackalloc char[FormatStringLength];
+            int charsWritten = Format(buffer);
+            return new string(buffer.Slice(0, charsWritten));
```

Comment:
Is `default` common enough that this should further check to see if it should return string.Empty instead of a new empty string?

Reply:
The string constructor will actually do that: if charsWritten is 0 here, this will end up giving back string.Empty.

Reply:
> ::blinks slowly:: really?

:smile:  Yeah.  If you look at string, you'll see that every constructor is an InternalCall, and then there's a corresponding method named Ctor that returns a string... when you new up a string, you're actually just calling that Ctor method under the covers, and those methods return string.Empty when the resulting string would be empty.